managing_providers/_topics/embedded_workflows.md

```diff
@@ -467,3 +544,49 @@ You can create a generic service catalog item that uses an embedded workflow. To
    The list of services and requests is shown when the catalog item is submitted. Clicking the request shows the execution status, including any embedded workflows.
 
    ![Workflow Status](../images/embedworkflow_runstatus.png)
+
+## Upgrading
```

Comment:
My only thought is this should probably go into the blog post and not the docs.  On downstream I could see it going into an Upgrade page.  For now though I'm ok with merging this.